frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -192,18 +192,20 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
     setNodes(formattedNodes);
   };
 
-  const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
-    return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
-    });
-  };
+  if(!node.label) {
```

Comment:
@parthdhanjal Why are you checking `node.label` here? It should be inside the `makeLabelNodesRequest` function and you should be specifically checking for label with value `cluster.ocs.openshift.io~1openshift-storage` as node can have multiple labels.